Sources/MusicPlayer/Players/SystemMedia.swift

```diff
@@ -164,4 +164,93 @@ private extension Notification.Name {
     static let mediaRemoteNowPlayingApplicationPlaybackStateDidChange = Notification.Name("kMRMediaRemoteNowPlayingApplicationPlaybackStateDidChangeNotification")
 }
 
+#elseif os(Linux)
+
+import playerctl
+
+extension MusicPlayers {
+    
+    public final class FakeSystemMedia: Agent {
+        
+        private let manager: UnsafeMutablePointer<PlayerctlPlayerManager>
+        
+        private var players: [MusicPlayers.MPRIS] = []
+        private var signals: [gulong] = []
+        
+        public init?(_: Void = ()) {
+            guard let manager = playerctl_player_manager_new(nil) else {
+                return nil
+            }
+            self.manager = manager
+            
+            super.init()
+            
+            GEventLoop.start()
+            
+            var playerNames: UnsafeMutablePointer<GList>? = playerctl_list_players(nil)
+            while (playerNames != nil) {
+                let playerName = playerNames!.pointee.data.assumingMemoryBound(to: PlayerctlPlayerName.self)
+                let player = playerctl_player_new_from_name(playerName, nil)
+                if player == nil {
+                    continue
+                }
+                playerctl_player_manager_manage_player(manager, player)
+                players.append(MusicPlayers.MPRIS(player: player!))
+                playerNames = playerNames!.pointee.next
+            }
+            
+            designatedPlayer = players.last { $0.playbackState.isPlaying } ?? players.last
```

Comment:
it won't change if user pause one player and resume another one right? why not using existing `MusicPlayers.NowPlaying`?